swift/ios-shoppe-demo/Views/cartTableViewController.swift

```diff
@@ -9,22 +9,31 @@
 import Foundation
 import UIKit
 
+var order: Order = Order()
+
 class CartTableViewController: UITableViewController {
 
-    var order: Order = Order()
     var items = [Product]()
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        print("\(order.cartOrderTotal())")
 
+        navigationItem.title = "Order Cart"
         navigationController?.navigationBar.tintColor = .white
-        tableView.register(UINib(nibName: "CartTableViewCell", bundle: nil), forCellReuseIdentifier: "cell")
-        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "cell2")
+        tableView.register(UINib(nibName: "CartTableViewCell", bundle: nil), forCellReuseIdentifier: "checkout")
+        tableView.register(UINib(nibName: "CartQuantityTableViewCell", bundle: nil), forCellReuseIdentifier: "quantity")
+        setOrderItemsForCart()
+    }
 
+    func setOrderItemsForCart() {
         items = order.items.filter { $0.quantity > 0 }
     }
 
+    func presentCheckout() {
+        let vc = CheckoutTableViewController()
```

Comment:
Is there a need for the `vc` assignment?  In `StoreViewController.swift` the first arg of `pushViewController` is a call to the constructor.

Reply:
Absolutely there is no need it can be one line 👍